src/Products/CMFPlone/profiles/dependencies/registry.xml

```diff
@@ -126,5 +126,13 @@
       <element key="plone-modal">{"actionOptions": {"displayInModal": false}}</element>
     </value>
   </record>
+  <records interface="Products.CMFPlone.controlpanel.browser.recyclebin.IRecycleBinControlPanelSettings"
+           prefix="plone-recyclebin"
+  >
+    <value key="recycling_enabled">True</value>
+    <value key="retention_period">30</value>
+    <value key="maximum_size">100</value>
+    <value key="restore_to_initial_state">False</value>
+  </records>
 
 </registry>
```

Comment:
We need an upgrade step in plone.app.upgrades to make sure that the changes in the profile folder will get applied to existing sites when they upgrade.